tests/test_pipeline.py

```diff
@@ -0,0 +1,84 @@
+from social_django.utils import load_backend, load_strategy
+
+import pytest
+
+from unicef_security.pipeline import create_unicef_user, get_username, social_details
+
+
+@pytest.fixture()
+def strategy():
+    return load_strategy()
+
+
+@pytest.fixture()
+def backend(strategy):
+    return load_backend(strategy=strategy, name='unicef-azuread-b2c-oauth2', redirect_uri='/')
+
+
+@pytest.fixture()
+def response():
+    return {
+        'id_token': 'token',
+        'token_type': 'Bearer',
+        'not_before': 1607541825,
+        'id_token_expires_in': 3600,
+        'profile_info': 'profile_info',
+        'scope': 'openid',
+        'access_token': 'access_token',
+        'exp': 1607545425,
+        'nbf': 1607541825,
+        'ver': '1.0',
+        'iss': 'https://tenant.b2clogin.com/1234567890/v2.0/',
+        'sub': 'abcdefgh', 'aud': 'abcdefgh',
+        'acr': 'b2c_1a_unicef_social_signup_signin',
+        'iat': 1607541825,
+        'auth_time': 1607541824,
+        'given_name': 'Given',
+        'family_name': 'Family',
+        'name': 'Given Family',
+        'idp': 'UNICEF Azure AD',
+        'email': 'test_undeliverable@unicef.org'
```

Comment:
I'd suggest making use of `@example.com` in tests

Reply:
and override the env var?

Reply:
You can pass in the `settings` param in the test and then override that specific setting e.g.
See https://pytest-django.readthedocs.io/en/latest/helpers.html#settings

Reply:
On the next todo: use the config file to try and get the settings from the project settings file and set defaults. 
As part of that work we can override the UNICEF address in the tests.